zipline/utils/sentinel.py

```diff
@@ -7,19 +7,33 @@
 
 
 def sentinel(name, doc=None):
-    @object.__new__   # bind a single instance to the name 'NotSpecified'
-    class result(object):
+    try:
+        value = sentinel._cache[name]  # memoized
+    except KeyError:
+        pass
+    else:
+        if doc == value.__doc__:
+            return value
+
+        raise ValueError(
+            'attempted to create sentinel with a used name and a different'
```

Comment:
I think this template is expecting the two docstrings, but you're templating in the name and the existing docstring.  How about this?

> New sentinel value %r conflicts with an existing sentinel of the same name.
Old Sentinel Docstring: %r
New Sentinel Docstring: %r
Resolve this conflict by changing the name of one of the sentinels.